documentation/modules/proc-exposing-kafka-bridge-service-local-machine.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-quickstart.adoc
+
+[id='proc-exposing-kafka-bridge-service-local-machine-{context}']
+= Exposing the Kafka Bridge service to your local machine
```

Comment:
This seems to be Strimzi operators specific as well.

Reply:
We can remove this and add it as a prereq. Maybe another one for the _Deploying_ guide

Reply:
removed + added as prereq for quickstart